src/core/src/op/roi_align.cpp

```diff
@@ -215,7 +338,8 @@ bool evaluate_roi_align(const HostTensorVector& args,
                          sampling_ratio,
                          spatial_scale,
                          pooling_mode,
-                         batch_indices->get_shape());
+                         batch_indices->get_shape(),
+                         aligned_mode);
     default:
         rc = false;
         break;
```

Comment:
Do you know why evaluate is in need here? If cpu plugin has the impl, and why not move it to evaluate_maps for template plugin?
And particularly to L353 bool op::v3::ROIAlign::evaluate, do you intention for v3 only?

Reply:
currently we didn't use evaluate function for ROIAlign_9, but keep unchanged for  ROIAlign_3. agree that to remove  evaluate function for ROIAlign_3 is functionally good. but I worry about that make modification on op's previous version part may bring in Inconsistency.

Hi, @mitruska , about 'to remove evaluate function for previous ROIAlign verion 3 ', do you have any suggestion?

Reply:
Usually we avoid big changes in existing ops to ensure backward compatibility, so I would suggest to leave `evaluate` for `v3::ROIAlign`. But as you have already discussed with Ilya, that `evaluate` is no longer needed for this op, it can be dropped for `v9::ROIAlign`.